### PR TITLE
Feat: Updates for Toggle isNonPeriodic addition

### PR DESCRIPTION
**Pull request by adewyer**

### CODE REVIEW

File: lang/en/docs/materials-designer/header-menu/edit.md [hunk 1]
```diff
@@ -27,3 +27,16 @@ The user can also `Reset` <i class="zmdi zmdi-close zmdi-hc-border"></i> all cha
...
+The `Edit` menu also allows the user to update the material ot use a `Conventional Cell`.
+
+## Toggle "isNonPeriodic"
+The 'Edit' menu also allows the user to update the material to be categorized as `non-periodic`. By clicking the `Toggle "isNonPeriodic"` button the materials lattice and basis will be updated so that it's coordinates are centered inside a cubic lattice with dimensionality based on the size of the structure. Upon saving the basis of the stucture will be set to Cartesian Units. the difference between a `periodic` and `non-periodic` structure is denoted by a change in the icon representing the structure. `Periodic` structures are denoted by <i class="zmdi zmdi-widget zmdi-hc-border"></i> icon while `non-periodic` structures are denoted by <i class="zmdi zmdi-device-hub zmdi-hc-border"></i> icons.
+
+## To Do
+1. Update image
+2. Create video for `Use Conventional Cell`
+3. Create vido for `Toggle "isNonPeriodic"`
+4. Image showing icon difference
```
Comment (adewyer) [inline lang/en/docs/materials-designer/header-menu/edit.md:42]: Remove lines 38-42.

File: lang/en/docs/materials-designer/header-menu/edit.md [hunk 2]
```diff
@@ -27,3 +27,16 @@ The user can also `Reset` <i class="zmdi zmdi-close zmdi-hc-border"></i> all cha
...
 
 <img data-gifffer="/images/materials-designer/edit-clone.gif" />
+
+## Use Conventional Cell
+
+The `Edit` menu also allows the user to update the material ot use a `Conventional Cell`.
+
+## Toggle "isNonPeriodic"
+The 'Edit' menu also allows the user to update the material to be categorized as `non-periodic`. By clicking the `Toggle "isNonPeriodic"` button the materials lattice and basis will be updated so that it's coordinates are centered inside a cubic lattice with dimensionality based on the size of the structure. Upon saving the basis of the stucture will be set to Cartesian Units. the difference between a `periodic` and `non-periodic` structure is denoted by a change in the icon representing the structure. `Periodic` structures are denoted by <i class="zmdi zmdi-widget zmdi-hc-border"></i> icon while `non-periodic` structures are denoted by <i class="zmdi zmdi-device-hub zmdi-hc-border"></i> icons.
+
```
Comment (adewyer) [inline lang/en/docs/materials-designer/header-menu/edit.md:37]: To Do:
1. update image
2. Do we need animations

File: lang/en/docs/materials-designer/source-editor/basis.md [hunk 3]
```diff
@@ -60,6 +60,15 @@ Click on the animation below to see the above in action. Here, we change the x-a
...
 <img data-gifffer="/images/materials-designer/ChangeMaterialBasis.gif" />
 
+
+## Non-Periodic Basis Transformations
+When [Edit - Toggle "isNonPeriodic"](../header-menu/edit.md) is used, the basis of the structure will automatically be converted to Cartesian Coordinates and translated to the center of the lattice. This is done by translating each basis coordinate so that the center of coordinates for the structure and the center point of the lattice align.
+
+## Toggle "isNonPeriodic" Animation
+
+Click on the animation below to see the **Toggle "isNonPeriodic" button in action.
+
```
Comment (adewyer) [inline lang/en/docs/materials-designer/source-editor/basis.md:70]: To Do: Add animation

File: lang/en/docs/materials/classification/non-periodic.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Non-Periodic Materials
+
+The complete **structure** of a **non-periodic chemcial system** is obtained from a [set of atoms](../../properties-directory/structural/non-periodic-basis.md), known as the **basis atoms**, that describe a set of non-repeating atoms that are held together by covalent or non-covalent interactions.  
+
+This results in a structure with a specific geometry which gives it a defined [point-group](../../properties-directory/structural/point-group.md)[^2] symmetry assignment.
+
+Examples of **non-periodic structures** include **molecules**[^1].
+## Visualization
+
```
Comment (adewyer) [inline lang/en/docs/materials/classification/non-periodic.md:9]: Add image

File: lang/en/docs/tutorials/materials/non-periodic.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+#### Toggling "isNonPeriodic"
+After the basis has been set, the user can then open the [Edit Menu](../../materials-designer/header-menu/edit.md) from the [Header Menu](../../materials-designer/header-menu/header-menu-intro.md) and select the `Toggle "isNonPeriodic` option. This will cause the basis and lattice of the structure to be immediately updated. The lattice size will be based on the size of the non-periodic structure you defined in the [Crystal Basis Editor](../../materials-designer/source-editor/basis.md). The shape of the lattice will be updated to Cubic. Finally the basis will be translated so that the center of the structure is aligned with the center of the lattice.
+
+#### Post Basis Editing
+If you wish to edit the basis after clicking `Toggle "isNonPeriodic"` then the basis will NOT automatically recenter itself until after the material is saved.
+
+#### Saving the Non-Periodic Structure
+Open the Input/Output Menu from the Header Menu. Choose the save option and select `submit`
+
+#### Animation
```
Comment (adewyer) [inline lang/en/docs/tutorials/materials/non-periodic.md:50]: Add Animation

File: lang/en/docs/tutorials/materials/non-periodic.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+With the material still selected, choose the <i class="zmdi zmdi-upload zmdi-hc-border"></i> icon for the upload tool in the top right actions toolbar.
+
+Once the material successfully uploads the **UPLOADED** column should show a <i class="zmdi zmdi-check zmdi-hc-border"></i> in the row of your material.
+
+#### Exit the Upload Files Explorer & Verify Material is listed in the Materials Explorer
+After the upload is successful you can exit out of the Upload Files Explorer using the top right X button. That should bring you back to the Materials Explorer Page.
+
+On the Materials Explorer Page you should see your material listed with a <i class="zmdi zmdi-device-hub zmdi-hc-border"></i> icon, and a <i class="zmdi zmdi-check zmdi-hc-border"></i> icon in the **Non-Periodic** column.
+
+#### Animation
```
Comment (adewyer) [inline lang/en/docs/tutorials/materials/non-periodic.md:100]: Add Animation

File: lang/en/docs/materials-designer/source-editor/basis.md (hunk 3 above)
Comment (adewyer) [reply lang/en/docs/materials-designer/source-editor/basis.md:70]: Created animation, need to convert to gif

**Comment by adewyer** [comment]
Added updates, still working on images/gifs. See Jira for drafts in comments.

**Comment by adewyer** [comment]
New Header Menu Edit image
edit-menu

File: lang/en/docs/materials-designer/header-menu/edit.md [hunk 7]
```diff
@@ -27,3 +27,11 @@ The user can also `Reset` <i class="zmdi zmdi-close zmdi-hc-border"></i> all cha
 The `Edit` menu also allows the user to `Clone` <i class="zmdi zmdi-collection-image zmdi-hc-border"></i> the currently selected structure into a new distinct entry in the items list sidebar. This feature is demonstrated in the short animation below, where we clone the original silicon structure into a new entry which appears below it:
 
 <img data-gifffer="/images/materials-designer/edit-clone.gif" />
+
+## Use Conventional Cell
+
+The `Edit` menu also allows the user to update the material ot use a `Conventional Cell`.
```
Comment (timurbazhirov) [inline lang/en/docs/materials-designer/header-menu/edit.md:33]: typo: "or use"

File: lang/en/docs/materials-designer/header-menu/edit.md [hunk 8]
```diff
@@ -27,3 +27,11 @@ The user can also `Reset` <i class="zmdi zmdi-close zmdi-hc-border"></i> all cha
...
+
+## Toggle "isNonPeriodic"
+The 'Edit' menu also allows the user to update the material to be categorized as `non-periodic`. By clicking the `Toggle "isNonPeriodic"` button the materials lattice and basis will be updated so that it's coordinates are centered inside a cubic lattice with dimensionality based on the size of the structure. Upon saving the basis of the stucture will be set to Cartesian Units. the difference between a `periodic` and `non-periodic` structure is denoted by a change in the icon representing the structure. `Periodic` structures are denoted by <i class="zmdi zmdi-widget zmdi-hc-border"></i> icon while `non-periodic` structures are denoted by <i class="zmdi zmdi-device-hub zmdi-hc-border"></i> icons.
```
Comment (timurbazhirov) [inline lang/en/docs/materials-designer/header-menu/edit.md:36]: I'd replace the last sentence starting with `Periodic` with a reference to https://github.com/Exabyte-io/documentation/pull/219/files#diff-287fca2a1961b4dc4531fdc461d08b6db12f024ad839edaedc49a8577ba6e9ffR28 instead to have one source of truth.

File: lang/en/docs/materials-designer/source-editor/basis.md [hunk 9]
```diff
@@ -60,6 +60,15 @@ Click on the animation below to see the above in action. Here, we change the x-a
 
 <img data-gifffer="/images/materials-designer/ChangeMaterialBasis.gif" />
 
+
+## Non-Periodic Basis Transformations
+When [Edit - Toggle "isNonPeriodic"](../header-menu/edit.md) is used, the basis of the structure will automatically be converted to Cartesian Coordinates and translated to the center of the lattice. This is done by translating each basis coordinate so that the center of coordinates for the structure and the center point of the lattice align.
+
```
Comment (timurbazhirov) [inline lang/en/docs/materials-designer/source-editor/basis.md:66]: We should refer to the change in lattice in https://github.com/Exabyte-io/documentation/pull/219/files#diff-19436c43ea51eedc5797131c91c17f24ac0330170499be4db6563c2e9a64b953R41 then

File: lang/en/docs/materials/actions/upload.md [hunk 10]
```diff
@@ -24,6 +24,9 @@ When large sets of files need to be imported at a single time, particular entrie
 
 Users may tag structures with a list of keywords. This is especially resourceful when large datasets of imported files need to be retrieved or filtered later. The general guidelines for entering tags and attaching them to entities are presented [here](../../entities-general/actions/metadata.md).
 
+#### Tagging Non-Periodic Structures
```
Comment (timurbazhirov) [inline lang/en/docs/materials/actions/upload.md:27]: Tagging a structure as Non-Periodic

File: lang/en/docs/properties-directory/structural/inchi-key.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
+# InChI
```
Comment (timurbazhirov) [inline lang/en/docs/properties-directory/structural/inchi-key.md:1]: InChiKey?

File: lang/en/docs/materials/classification/non-periodic.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
+# Non-Periodic Materials
+
+The complete **structure** of a **non-periodic chemcial system** is obtained from a [set of atoms](../../properties-directory/structural/non-periodic-basis.md), known as the **basis atoms**, that describe a set of non-repeating atoms that are held together by covalent or non-covalent interactions.  
+
+This results in a structure with a specific geometry which gives it a defined [point-group](../../properties-directory/structural/point-group.md)[^2] symmetry assignment.
+
+Examples of **non-periodic structures** include **molecules**[^1].
+## Visualization
```
Comment (timurbazhirov) [inline lang/en/docs/materials/classification/non-periodic.md:8]: Missing newline after line 7

File: lang/en/docs/materials/classification/non-periodic.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
+# Non-Periodic Materials
```
Comment (timurbazhirov) [inline lang/en/docs/materials/classification/non-periodic.md:1]: So are we introducing materials classification? If yes, we should create an overview file at that level - `materials/classification/overview.md` and refer to the included content from there. Periodic Materials should also be described. And maybe a reference to materials sets.

Comment (adewyer) [reply lang/en/docs/materials/classification/non-periodic.md:1]: Created a new overview file.  https://github.com/Exabyte-io/documentation/pull/219/files#diff-85aac972bf958fff2b551016debab75eb161ff16bc3ae5ce2bc0bce5bd546bd9

File: lang/en/docs/tutorials/materials/non-periodic.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
+# Create Non-Periodic Structures
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/non-periodic.md:1]: I think we don't need a tutorial on how to create a non-periodic structure. Adding content into the corresponding materials actions pages would be sufficient.

**Comment by adewyer** [comment]
updated gif before adding to git-lfs

toggle-is-non-periodic-button

File: lang/en/docs/materials/overview.md [hunk 15]
```diff
@@ -30,7 +30,7 @@ The [Data](data.md) section contains an example of JSON representation of a mate
 
 ## Classification
 
-Materials are distinguished based on whether they are [crystalline](classification/crystalline.md) or [non-crystalline](classification/non-crystalline.md).
+Materials are distinguished based on whether they are [crystalline](classification/crystalline.md), [non-crystalline](classification/non-crystalline.md), or [non-periodic](classification/non-periodic.md). An overview of classifications can be found [here](classification/overview.md)
```
Comment (timurbazhirov) [inline lang/en/docs/materials/overview.md:33]: We don't really support non-crystalline materials, so let's just clean up all that related stuff, please. Should be just periodic and non-periodic structures.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**